### DISCUSSION TITLE
No way to sort database properties in any other way than alphabetical

### DISCUSSION THREAD

**Discussion by mleone87** [ramnes/notion-sdk-py#146] [Q&A]
Hello,

am I wrong or there is no way to sort database properties in an arbitrary way with the APIs? In the GUI is possible to it in any form and by default properties are attached on the right of existing one.
Can't see a way to do it with the APIs.

thanks!

**Comment by ramnes** [comment]
Does that help?

https://developers.notion.com/reference/post-database-query-sort

**Reply by mleone87** [reply]
I think that this sorts database rows, not properties(coloums)?

I guess that this feature is not yet present in databases api, like creating new views.

**Reply by ramnes** [reply]
Ah, I get it now. :)

Indeed I don't think it's possible with the current state of the API, I'm sorry.
